polling_stations/apps/data_finder/helpers/geocoders.py

```diff
@@ -39,12 +39,22 @@ def geocode(self):
             raise PostcodeError("No location information")
 
         local_auth = geocoder.get_code("lad")
-        error_values = [
+        lad_error_values = [
             "L99999999",  # Channel Islands
             "M99999999",  # Isle of Man
             "",  # Terminated Postcode or other
         ]
-        if not local_auth or local_auth in error_values:
+        usertype = geocoder.get_code("usertype")
+        exclude_prefixes = [
+            "GIR",  # Banks
+        ]
+        if (
+            not local_auth
+            or local_auth in lad_error_values
+            or usertype == "1"  # large users
+            or self.postcode[:2]
+            in exclude_prefixes  # things we don't want to geocode from onspd
+        ):
```

Comment:
`self.postcode[:2]`
is the first 2 characters
but `GIR` is 3 characters
Given the Girobank postcode is a large user anyway, I think we could just completely ditch this specific check.